docs/guides/minio-image-upload.md

```diff
@@ -16,31 +15,31 @@ Minio Guide for CodiMD
    *Please notice this is not for productive use as all your data gets lost
    when you stop this container*
 
-2. Next step is to get the credentials form the container:
+1. Next step is to get the credentials form the container:
 
    ```sh
    docker logs test-minio
    ```
 
    ![docker logs](../images/minio-image-upload/docker-logs.png)
 
-3. Open http://localhost:9000 and login with the shown credentials.
+1. Open <http://localhost:9000> and login with the shown credentials.
 
    ![minio default view](../images/minio-image-upload/default-view.png)
 
-4. Create a bucket for CodiMD
+1. Create a bucket for CodiMD
 
    ![minio create bucket](../images/minio-image-upload/create-bucket.png)
 
-5. Add a policy for the prefix `uploads` and make it read-only.
+1. Add a policy for the prefix `uploads` and make it read-only.
```

Comment:
Am I the only one who does not care for `1.`-everywhere for ordered lists?

I know why it is done, but I think it partly defeats the purpose of Markdown - being well-readable even without a renderer.

As before, I'd love to hear other comments, of course, and will gladly shut up if I'm the only one with that particular opinion.

Reply:
(same comment applies to all files where numbered lists were swapped to be `1.` lists)

Reply:
totally agree. making every entry `1.` defeats the purpose. Starting them with `1.` and having the correct followup numbers would be great though. Can that be done with remark?

Reply:
https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-ordered-list-marker-value

this should be: [`remark-preset-lint-markdown-style-guide: ordered`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-ordered-list-marker-value), the readme claims that this would be the default, but our base ruleset [`preset-lint-markdown-style-guide`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-preset-lint-markdown-style-guide#rules) overrides this.

Reply:
See the [documentation about this rule](https://unifiedjs.com/explore/package/remark-lint-ordered-list-marker-value/).

The default value is `one`:

> When set to 'one', bullets should always be 1.

I think you'd like to set it to `ordered`?

I have a split opinion about the default value of `one`. The cons is that it makes the markdown less readable. But the pro is that it is very easy to maintain as you don't have to renumber the list when adding or removing an item from the list.

I personally prefer the `one` value because it is the best solution to avoid inconsistency in lists.

Reply:
Their readme is somewhat weirdly structured, I also first thought it was `one` by default, but it's actually `ordered`:

![grafik](https://user-images.githubusercontent.com/124909/86267412-f4035800-bbc6-11ea-93ca-db5f133dfbc5.png)

The table below that lists the value that's overridden inside `remark-preset-lint-markdown-style-guide` (which we're basing off of).

Reply:
There is the default value of the rule, and the default value of the preset. Both default values are not the same.

Reply:
yes, this is exactly what I wrote here: https://github.com/codimd/server/pull/410#discussion_r448458446

Reply:
I'm strongly against `one` as our devs and possibly our (admin) users will see these files in a plaintext editor rather than with a markdown preview.

Reply:
I've added 2 commits to change `ordered-list-marker-value` from `one` to `ordered`. I can now squash or not the commits depending on the final decision about this default value.

I just want to repeat that I personally prefer the one value because it is the best solution to avoid inconsistency in list numering. There is already some inconsistency, such as the [keycloak page](/codimd/server/blob/master/docs/guides/auth/keycloak.md) where item 6 follows item 4 without any item 5:

> 4. Go to the client you set up in the previous steps using the Clients page, then choose the Client Scopes tab. Apply the scope you've created. This should mitigate errors as seen in [codimd/server#56](https://github.com/codimd/server/issues/56), as the `/userinfo` endpoint should now bring back the user's ID under the `id` key as well as `sub`.
> 
> ---
> 
> 6. In the `docker-compose.yml` add the following environment variables to `app:` `environment:`

This manual renumbering is also a pain, and is no more a pain when  `ordered-list-marker-value` is set to `one`.